statsmodels/tools/rng_qrng.py

```diff
@@ -0,0 +1,50 @@
+import numbers
+
+import numpy as np
+import scipy.stats as stats
+
+
+def check_random_state(seed=None):
+    """Turn `seed` into a random number generator.
+
+    Parameters
+    ----------
+    seed : {None, int, `numpy.random.Generator`,
```

Comment:
Generator also support Sequence[int].

Reply:
`Sequence[int]` did not work, only `Sequence`. But I guess it's fine as it will be caught by NumPy. Otherwise we could just put `np.random.default_rng(seed)` at the end to catch all errors.